modules/tools/migration/src/main/java/org/wso2/mb/migration/DBConnector.java

```diff
@@ -225,6 +267,203 @@ void writeMessageRouter(String routerName, String routerData) throws SQLExceptio
             conn.close();
         }
     }
+
+    /**
+     * Make queue names of MB_QUEUE table all simple letters. Before that read and delete
+     * all entries in MB_BINDING to get rid of foreign key constraints. Then update
+     * MB_BINDING table as well with modified queue names.
+     *
+     * @throws SQLException in case of executing updates
+     */
+    void updateQueueNamesInQueuesAndBindings() throws SQLException {
+        getConnection();
+        try {
+            PreparedStatement getBindingsStatement = conn.prepareStatement(GET_BINDINGS);
+            ResultSet bindingsResultSet = getBindingsStatement.executeQuery();
+
+            //delete all bindings to get rid of constraints
+            PreparedStatement removeBindingsStatement = conn.prepareStatement(DELETE_ALL_BINDINGS);
+            removeBindingsStatement.executeUpdate();
+
+            updateQueueNamesInQueues();
+
+            while (bindingsResultSet.next()) {
+                String queueName = bindingsResultSet.getString(QUEUE_NAME);
+                String bindingData = bindingsResultSet.getString(BINDING_DETAILS);
+                if (queueNameHasCapitals(queueName)) {
+                    String newQueueName = queueName.toLowerCase();
+                    bindingData = bindingData.replaceAll(queueName, newQueueName);
+                    queueName = newQueueName;
+                }
+
+                PreparedStatement addBindingsStatement = conn.prepareStatement(INSERT_BINDING);
+                addBindingsStatement.setString(1, bindingsResultSet.getString(EXCHANGE_NAME));
+                addBindingsStatement.setString(2, queueName);
+                addBindingsStatement.setString(3, bindingData);
+
+                addBindingsStatement.executeUpdate();
+
+            }
+
+        } catch (SQLException e) {
+            e.printStackTrace();
```

Comment:
Please ensure that we throw the exception here ? e.printStackTrace() will not work as this will not be included in the log

Reply:
Hi Pamod,
This was fixed with following pr - https://github.com/wso2-attic/product-mb/pull/511

Reply:
Hi Pamod,
This was fixed with following pr - #511